File: androidapp/src/com/chyvonalu/findnumimg/androidapp/MyFragment.java
package com.chyvonalu.findnumimg.androidapp;

import android.support.v4.app.Fragment;

public class MyFragment extends Fragment {

	public MainActivity getMainActivity() {
		return (MainActivity) getActivity();
	}
	public void onEnter() {}

	public void onLeave() {}
}

File: androidapp/src/com/chyvonalu/findnumimg/androidapp/MyFragment.scala
package com.chyvonalu.findnumimg.androidapp

import android.support.v4.app.Fragment

trait MyFragment extends Fragment with HasLogger with FragmentLifeCycleLogging {
  val TAG: String

  def getMainActivity: MainActivity = {
    return getActivity.asInstanceOf[MainActivity]
  }


  def onEnter {
  }

  def onLeave {
  }
}